pkg/webhook/admission/deployment/model_provisioning_injector.go

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2019 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployment
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kubeflow/kfserving/pkg/constants"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	ModelProvisioningContainerName    = "model-provisioner"
+	ModelProvisioningVolumeName       = "kfserving-provision-location"
+	ModelProvisioningContainerImage   = "kcorer/kfdownloader"
```

Comment:
Let's make this configurable & a standardized link. Just cut me an issue to setup to automated build.

Reply:
made it standard... trying to figure out how to make it configurable.. the webhook doesn't load config right now.. can point me to an example of how to do this correctly?